src/cljs/main/broadfcui/common/gcs_file_preview.cljs

```diff
@@ -27,19 +27,21 @@
                         [:div {:style {:display "inline-block" :width 185}} (str label ": ")]
                         contents])
              data-empty (or (= data-size "0") (string/blank? data-size))
-             bam? (re-find #"\.ba[mi]$" (:object props))]
+             bam? (re-find #"\.ba[mi]$" (:object props))
+             img? (re-find #"\.jpe?g|png|gif|bmp$" (:object props))
```

Comment:
opinions wanted: given that `bam?` and `img?` are only used in the `hide-preview?` below, should these be collapsed into one regex?